tests/CMakeLists.txt

```diff
@@ -174,26 +174,14 @@ set_tests_properties(test_log_negative
 # Try building with various cmake options
 add_test(NAME test_build_all_off
         COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options.sh
-        ${PROJECT_SOURCE_DIR} test_build_all_off "-DBUILD_OSTREE=OFF -DBUILD_DEB=OFF")
+        ${PROJECT_SOURCE_DIR} test_build_all_off "-DBUILD_SYSTEMD=off")
 
 # Build with everything. BUILD_OPCUA requires boost::filesystem::relative,
 # which is not in Ubuntu 16.04, so disable it for now
 add_test(NAME test_build_all_on
         COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options.sh
         ${PROJECT_SOURCE_DIR} test_build_all_on "-DBUILD_WITH_CODE_COVERAGE=ON -DBUILD_OSTREE=ON -DBUILD_DEB=ON -DBUILD_SOTA_TOOLS=ON -DBUILD_ISOTP=ON -DBUILD_LOAD_TESTS=ON")
 
-add_test(NAME test_build_debug
-        COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options.sh
-        ${PROJECT_SOURCE_DIR} test_build_debug "-DCMAKE_BUILD_TYPE=Debug")
-
-add_test(NAME test_build_release_no_ostree
-        COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options.sh
-        ${PROJECT_SOURCE_DIR} test_build_release_no_ostree "-DCMAKE_BUILD_TYPE=Release -DBUILD_OSTREE=off")
-
-add_test(NAME test_build_release
-        COMMAND ${PROJECT_SOURCE_DIR}/tests/build_with_options.sh
-        ${PROJECT_SOURCE_DIR} test_build_release "-DCMAKE_BUILD_TYPE=Release -DBUILD_OSTREE=on")
-
```

Comment:
Hmm, my other comment disappeared, I guess since it was connected to a specific commit. Oh well. Point is: I think I'm okay with getting rid of these, although they do more or less provide some use while testing locally.

Reply:
Yeah, sorry about nuking your comment :). Let's check with @cajun-rat maybe